pkg/auth/dependency/webapp/translation.go

```diff
@@ -43,8 +43,8 @@ var TemplateAuthUITranslationJSON = template.Spec{
 	"sso-login-id-separator": "or",
 
 	"phone-number-placeholder": "Phone number",
-	"login-id-placeholder": "Email or username",
-	"use-text-login-id-description": "Use email or username instead",
+	"login-id-placeholder": "{0, select, email {email} username {username} other {email or username}}",
```

Comment:
Across the whole auth ui, the first character of the placeholder should be upper case. Please update it for consistency.

Reply:
Fixed.